### PR TITLE
Correctly manage maven-release-plugin on pom.xml

### PR DESCRIPTION
**Pull request by fbelzunc**
@reviewbybees @jtnord

### CODE REVIEW

**Comment by lordofthejars** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -88,9 +88,6 @@
         <groupId>org.apache.maven.plugins</groupId>
```
Comment (jtnord) [inline pom.xml]: this whole plugin section should go in `<pluginManagement>`

**Comment by alecharp** [comment]
it's bad to have to do that but :bee: as it is because of the parent-pom configuration, not because of the plugin configuration.

**Comment by jtnord** [comment]
you moved the findbugs - but not the m-r-p?

**Comment by alecharp** [comment]
The two plugins were inside the `<pluginManagement/>` before. @fbelzunc followed my advices to move m-r-p outside of this section because of the parent pom

**Comment by fbelzunc** [comment]
@alecharp Adding inside `pluginManagement` because otherwise when releasing you are not working inside the build lifecycle. Inside `pluginManagement` it will be valid inside build lifecycle and command line.

**Comment by jtnord** [comment]
:bee:

**Pull request merged**